### DISCUSSION TITLE
初始脚本创建的Apollo用户无法登录

### DISCUSSION THREAD

**Discussion by zisongsun** [apolloconfig/apollo#4455] [Q&A]
使用最新版本的Mysql脚本创建了User：apollo之后，无法使用apollo/admin进行登录。
界面报错：incorrect username or password
使用的sql：
INSERT INTO `Users` (`Username`, `Password`, `UserDisplayName`, `Email`, `Enabled`)
VALUES
    ('apollo', '$2a$10$7r20uS.BQ9uBpf3Baj3uQOZvMVvB1RN3PYoKE94gtz2.WAOuiiwXS', 'apollo', 'apollo@acme.com', 1);

**Comment by nobodyiam** [comment]
Duplicate of #4456
